src/main/webapp/app/entities/participant/participant.module.ts

```diff
@@ -13,12 +13,23 @@ import {
     ParticipantDeleteDialogComponent,
     participantRoute,
     participantPopupRoute,
+    ParticipantSettingsComponent,
+    participantSettingsRoute,
+    participantStudiesRoute,
+    ParticipantStudiesComponent,
+    ParticipantStudiesOptOutComponent,
+    ParticipantStudiesOptOutPopupComponent,
+    participantStudiesPopupRoute,
+    ParticipantOptOutPopupService,
 } from './';
 import { ParticipantPipe } from './participant.pipe';
 
 const ENTITY_STATES = [
     ...participantRoute,
     ...participantPopupRoute,
+    participantSettingsRoute,
+    participantStudiesRoute,
```

Comment:
are the "..."'s needed for these lines here? Not too sure what they do though

Reply:
I think the "..." are used for something that has multiple ROUTES inside it. Because my ones only create one ROUTE each, I didn't need the "...", but I think they are still needed for the rest.

Reply:
Sweet as, that makes sense